<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paciente</class>
 <widget class="QDialog" name="paciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ver_examen">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver examen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
